### PR TITLE
Added Chopsticks

### PR DESCRIPTION
**Pull request by Redfire1331**
## About the PR
i added chopsticks!
they can be obtained in the dinnerware and mr.chang vendors
## Why / Balance
chopsticks are cool!

## Technical details
added chopsticks, they work like a fork

## Media

https://github.com/space-wizards/space-station-14/assets/125223432/6c3d7f76-58c4-403f-b427-396876e1f8cf

 

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes
none

**Changelog**
Added chopsticks! now available in dinnerware and Mr.Chang vendors!

### CODE REVIEW

File: Resources/Prototypes/Entities/Objects/Misc/chopsticks.yml
Comment (rosieposieeee) [inline Resources/Prototypes/Entities/Objects/Misc/chopsticks.yml:1]: name field should not be capitalized

**Comment by Ghagliiarghii** [comment]
I really want this to have like 6 piercing damage when swung / thrown...

File: Resources/Prototypes/Entities/Objects/Misc/chopsticks.yml
Comment (Redfire1331) [reply Resources/Prototypes/Entities/Objects/Misc/chopsticks.yml:1]: done!

**Comment by metalgearsloth** [comment]
Are these chopsticks for giants, it's bigger than the mob.

It doesn't need to be accurate but it shouldn't occupy 32x32.

**Comment by Redfire1331** [comment]
> Are these chopsticks for giants, it's bigger than the mob.
> 
> It doesn't need to be accurate but it shouldn't occupy 32x32.

I’ll change the icon tomorrow

File: Resources/Prototypes/Entities/Objects/Misc/chopsticks.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+- type: entity
+  parent: BaseItem
+  id: ChopSticks
+  name: chop sticks
```
Comment (lzk228) [inline Resources/Prototypes/Entities/Objects/Misc/chopsticks.yml:4]:
```suggestion
  name: chopsticks
```

**Comment by Redfire1331** [comment]
> Are these chopsticks for giants, it's bigger than the mob.
> 
> It doesn't need to be accurate but it shouldn't occupy 32x32.
chopstick redesign
is this a good size?

**Comment by Redfire1331** [comment]
should i add the chopsticks to the mrchangs vendor?

**Comment by Redfire1331** [comment]
> Are these chopsticks for giants, it's bigger than the mob.
> 
> It doesn't need to be accurate but it shouldn't occupy 32x32.

fixed

**Comment by UbaserB** [comment]
would it be possible to add a chopsticks case as well? So i can carry it around me

**Comment by Redfire1331** [comment]
> I was really hoping using the chopsticks would have a satisfying snap sound to pull them apart before you can use them to eat

good idea ill try making that

**Comment by Redfire1331** [comment]
> I was really hoping using the chopsticks would have a satisfying snap sound to pull them apart before you can use them to eat

@Plykiya  Added!

**Comment by Plykiya** [comment]
> > I was really hoping using the chopsticks would have a satisfying snap sound to pull them apart before you can use them to eat
> 
> @Plykiya Added!

you're my fucking hero

**Pull request merged**